Update 14 (#276)
NilPointer-Software/bdfd-wiki: src/bdscript/randomGuildID.md

# $randomGuildID

> It's returns a random guild ID that the bot is in.

Reviewer: ```suggestion
Returns a random guild ID from servers the bot is in.
```
